src/Http/Routing/WebRouting.php

```diff
@@ -125,6 +132,14 @@ public function wire(): void
                 $this->router->get("/invitation", "create")->name("invitation");
                 $this->router->post("/invitation", "store")->name("invitation.store");
             });
+
+            $this->router->controller(UserSettingsController::class)->group(function (): void {
+                $this->router->get("/settings", "index")->name("settings");
+                $this->router->post("/settings", "enable2Fa")->name("enable2fa");
+                $this->router->post("/settings/disable", "disable2Fa")->name("disable2fa");
+                $this->router->get("/settings/disable", "disable2FaConfirm")->name("disable2fa.confirm");
+                $this->router->post("/settings/disabled", "disable2FaStore")->name("disable2fa.store");
+            });
```

Comment:
Right now only a user with the Admin role can access these routes. You must move it out of the Admin middleware group.